src/main/resources/SLING-INF/nodetypes/nodetypes.cnd

```diff
@@ -28,6 +28,6 @@
 <pant = 'http://redhat.com/pantheon'>
 
 // Base type for modules
-[pant:module] > nt:unstructured, sling:Folder
+[pant:module] > nt:unstructured
```

Comment:
I just don't think we need it. I'm also trying to keep our node definitions as simple as possible.

Reply:
Hmm, ok. Joseph noticed some weird behavior this morning with the latest commit. I wonder if this might have something to do with it. (Just speculation for now).